lib/Graph/Nodes.cpp

```diff
@@ -591,6 +591,18 @@ void SparseLengthsWeightedSumNode::verify() const {
          "Weights and Indices must have the same size");
 }
 
+void LengthsToRangesNode::verify() const {
+  assert(getResult().getElementType() == getLengths().getElementType() &&
+         "Mismatched element types");
+  assert(getLengths().getElementType() == ElemKind::Int64ITy &&
```

Comment:
will it be int32ity eventually? (if so maybe add TODO, so that we could fix it real quick later).